src/Components/test/E2ETest/ServerRenderingTests/FormHandlingTests/FormWithParentBindingContextTest.cs

```diff
@@ -896,6 +896,19 @@ public void FormNoAntiforgeryReturnBadRequest(bool suppressEnhancedNavigation)
         DispatchToFormCore(dispatchToForm);
     }
 
+    [Fact]
+    public void CanUseAntiforgeryTokenInWasm()
+    {
+        var dispatchToForm = new DispatchToForm(this)
+        {
+            Url = "forms/antiforgery-wasm",
+            FormCssSelector = "form",
+            InputFieldId = "Value",
+            SuppressEnhancedNavigation = true,
+        };
+        DispatchToFormCore(dispatchToForm);
+    }
```

Comment:
Could there be a test to show this works on InteractiveServer too? I know it's not such a mainstream case but if we're going to implement support for it (and I think you have done), people will use it, and then we have to be sure not to regress it.

Reply:
There's no point on doing this for server, even if you can get the request token, you can't get the cookie token as your HttpClient instance runs on the server, not on the browser, where the cookie is set.

We should provide guidance on limiting use HttpClient to WebAssembly only and recommend going through the APIs directly on Server.

We can (if we want) flow the cookie token to the Server to make this work, but that is messy, IMO.

Reply:
Then we should actively stop it from going into the persisted state for server, right? Otherwise people will certainly use it.

If I'm misunderstanding and it's already *not* going into persisted state for server, please let me know.

Reply:
Yes, I've changed it to only go to webassembly.

Reply:
Great!